### ISSUE TITLE
[Usage]: 怎么得到每一个输出token的logits，而不是softmax之后的logprob？

### ISSUE THREAD

**Issue by TonyUSTC** [vllm-project/vllm#10688]
### Your current environment

我需要softmax之前的logits，但是目前vllm好像返回不了logits，huggingface transformer是可以的。请问vllm怎么实现呢？

### Before submitting a new issue...

- [X] Make sure you already searched for relevant issues, and asked the chatbot living at the bottom right corner of the [documentation page](https://docs.vllm.ai/en/latest/), which can answer lots of frequently asked questions.

**Comment by TonyUSTC** [comment]
我修改了下model_executor/layers/sampler.py，在Sample the next tokens之后，重置logprobs = logits。不够优雅，但是推理结果可以返回logits了
![image](https://github.com/user-attachments/assets/4badbb69-6738-4d31-bcbc-ebb069650e86)

**Comment by youkaichao** [comment]
cc @DarkLight1337 can you give a usage example and then close the issue?
